### ISSUE TITLE
The connection state indicator is not intuitive/clear

### ISSUE THREAD

**Issue by nirinchev** [realm/realm-studio#1078]
As a user it's not clear to me what the connection state indicator means and what I should do as a user if it's not green. Particularly, when it goes red and says "disconnected from ...", it's not clear what I can do to resolve it - is Studio going to keep retrying the connection in the background or should I close all windows and go through the connect to server flow again? I feel like we should show some sort of a hint when users hover over it to give them common causes for problems and guidance on how to proceed.

**Comment by kraenhansen** [comment]
I agree. I was thinking that the latest error could be shown when hovering to get an idea on why the client disconnected. I wanted to write "Reconnecting in {seconds}" as a countdown, but unfortunately it's not trivial to calculate when the client will be reconnecting again .. but I agree it should be obvious to the user that Studio will eventually be reconnecting.

**Comment by bmunkholm** [comment]
Kibana e.g. uses this in the top:
![image](https://user-images.githubusercontent.com/1215687/51479114-a9549e00-1d8d-11e9-8854-178e5c26309b.png)

Then clicking More gives details:
![image](https://user-images.githubusercontent.com/1215687/51479150-c25d4f00-1d8d-11e9-920d-84e56818bb22.png)

Which is this case however isn't helpful :-)
